addon/components/mdc-menu.js

```diff
@@ -28,6 +28,11 @@ export default Ember.Component.extend(MDCComponent, {
    * @type {?Function}
    */
   cancel: x => x,
+  /**
+   * Enabling this is generally a bad idea for accessibility, so think hard about using this attribute.
```

Comment:
This comment seems a little awkward for what the attribute is called. When you say "enabling this," you mean "disabling focus" which seems a little backwards to me. :shrug:

Reply:
I guess I'd recommend changing the comment to say "Disabling focus is generally a bad idea [...]"

Reply:
Enabling = setting this to `true` = what would be generally bad for accessibility

Reply:
yeah but setting to true means disabling. so you're saying enabling the disabling is bad for accessibility. Idk, just sounds weird to me but that could just be me.

Reply:
I agree that using the word "enabling" to talk about disabling could be confusing. Better safe (explicit) than sorry (causing misunderstandings that lead to bugs)!